modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,379 @@
+* xref:manage:remote-read-replicas.adoc[Remote read replicas] and xref:manage:topic-recovery.adoc[topic recovery] are not supported for Iceberg-enabled topics.
+
+== Architecture 
+
+Apache Iceberg is an open source format specification for defining structured tables in a data lake. The table format lets you quickly and easily manage, query, and process huge amounts of structured and unstructured data. This is similar to the way in which you would manage and run SQL queries against relational data in a database or data warehouse. The open format lets you use many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is referred to as a _data lakehouse_.
+
+In the Iceberg specification, tables consist of the following layers:
+
+* Data layer
+** Data files: Store the data. The Iceberg integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale. They come with compression capabilities that optimize files for object storage. 
+* Metadata layer
++
+--
+** Manifest files: Track data files and contain metadata about these files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. The Iceberg integration supports two <<set-up-catalog-integration,catalog integration>> types. You can configure Redpanda to catalog files stored in the same object storage bucket or container where the Iceberg data files are located, or you can configure Redpanda to use an https://iceberg.apache.org/concepts/catalog/#catalog-implementations[Iceberg REST catalog^] endpoint to update an externally-managed catalog when there are changes to the Iceberg data and metadata.
+
+image::shared:iceberg-integration.png[]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage. The brokers also write events to segment files stored on disk. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through many compatible https://iceberg.apache.org/vendors/[data lakehouse, data engineering, and business intelligence tools]. 
```

Comment:
> The brokers also write events to segment files stored on disk.

This feels like it's wedged in here and I don't know if this statement is meaningful to an end user. Does the average end user know what a segment is? I think we're trying to say that the iceberg data is in addition to the current log oriented format that support normally and doesn't replace it.